src/components/organisms/Header.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import classNames from 'classnames';
+
+// images
+import logoWhite from '../../images/logoWhite.svg'
+import logoBlack from '../../images/logoBlack.svg'
+
+// Components
+import Menu from '../molecules/Menu';
+
+const Header = ({ dark }) => {
+  const headerClass = classNames({
+    header: true,
+    'header-dark': dark
+  });
+  return (
+    <header className={headerClass}>
+      <a href=""><img src={(dark && logoWhite) || logoBlack} alt="" /></a>
```

Comment:
It will be good if we specify the alternative text of the images for a better accessibility.
```suggestion
      <a href=""><img src={(dark && logoWhite) || logoBlack} alt="Neobit Logo" /></a>
```

Reply:
Ready